tfm/targets/TARGET_ARM_SSG/TARGET_MUSCA_S1/device/platform_base_address.h

```diff
@@ -0,0 +1,163 @@
+/*
```

Comment:
This file and `tfm_peripherals_def.h` are copied by `tfm_ns_import.yaml` - the user has to run the scripts anyway. I'd suggest removing the device target files and adding the paths to `.gitignore`.

Reply:
The files are checked in for `MUSCA_B1`. The reason they are checked, because they give the maintainer an overview about the flash base and address, because if they are changed it may impact the linker scripts, following further changes to a specific target.